common/src/main/java/com/microsoft/identity/common/internal/cache/SharedPreferencesFileManager.java

```diff
@@ -90,6 +90,20 @@ public static SharedPreferencesFileManager getSharedPreferences(final Context co
         return cachedFileManager;
     }
 
+    /**
+     * Constructs an instance of SharedPreferencesFileManager. Operating mode is always MODE_PRIVATE.
+     *
+     * @param context Interface to global information about an application environment.
+     * @param name The desired {@link android.content.SharedPreferences} file. It will be created if it does not exist.
+     * @param storageHelper The {@link IStorageHelper} to handle encryption/decryption of values.
+     * @return The SharedPreferencesFileManager instance.
+     */
+    public static SharedPreferencesFileManager getSharedPreferences(final Context context,
```

Comment:
nit: If we are only going to use this to getSharedPreferences.. we can mark the other one (line78) as deprecated

Reply:
15b27a5ca2fa926df66223b7f9380c30b810b20c